lib/maze/configuration.rb

```diff
@@ -75,6 +75,9 @@ def initialize
     # Enables awareness of a public IP address on Buildkite with the Elastic CI Stack for AWS.
     attr_accessor :aws_public_ip
 
+    # Disables timestamp validation on spans
+    attr_accessor :span_timestamp_validation
```

Comment:
We'll need to make sure this is set to `true` in one of the `Before` hooks, so it doesn't bleed over into subsequent scenarios.